.travis.yml

```diff
@@ -0,0 +1,121 @@
+sudo: required
+dist: precise
```

Comment:
precise is ubuntu 12.04, trusty is 14.04; the reason why you needed that gettext hack is that you've chosen the wrong repository